
let resultado = null;

// console.log(resultado.toString());

try {
    console.log(resultado.toString());
} catch (erro) {
    console.log(`Houve um erro: ${erro}`);
}

console.log('Programa continuando...');
